website/docs/components/catalogs/databricks.md

```diff
@@ -53,6 +55,7 @@ The `params` field is used to configure the connection to the Databricks Unity C
 - `mode`: The execution mode for querying against Databricks. The default is `spark_connect`. Possible values:
   - `spark_connect`: Use Spark Connect to query against Databricks. Requires a Spark cluster to be available.
   - `delta_lake`: Query directly from Delta Tables. Requires the object store credentials to be provided.
+  - `sql_warehouse`: Use SQL Statement Execution API to query against a Databricks SQL Warehouse.
```

Comment:
@kczimm let's provide some help to the user where to find these parameters and examples of each.